Domains/2-DisciplinePhysical/StructuralPhysical/StructuralPhysical.remarks.md

```diff
@@ -7,8 +7,90 @@ remarksTarget: StructuralPhysical.ecschema.md
 
 This schema contains classes that are used to model the real-world physical entities that comprise the structural systems of infrastructure.
 
-## StructuralPhysicalModel
+## Entity Classes
+
+### StructuralPhysicalModel
 
 ***This class has been deprecated. Do not use.***
 
-Applications, services and components should never create instances of this class. Use `PhysicalModel` instead.
\ No newline at end of file
+Applications, services and components should never create instances of this class. Use `PhysicalModel` instead.
+
+### StructuralElement
+The abstract class `StructuralElement` is the generalization of structural member.
+
+Equivalent to [IfcStructuralItem](https://standards.buildingsmart.org/IFC/RELEASE/IFC4_3/HTML/lexical/IfcStructuralItem.htm).
+
+### StructuralMember
+The class `StructuralMember` is the base class of all structural items. This class is not abstract and could be used to define any genral element, but it is recommended to use any defined subclass if available.
+
+Classification of `StructuralMember` instances can be achieved via instances of `StructuralMemberType`.
+
+`StructuralMember` should be used as a base of any spetialized class.
+
```

Comment:
We've been also explicit about some details like what kind of model instances of a given class are meant to be in. e.g. `StructuralMember` instances must be contained in PhysicalModels.